workers/src/main/scala/workers/ItcServer.scala

```diff
@@ -50,7 +51,11 @@ object ItcServer extends WorkerServer[IO, ItcMessage.Request] with ItcPicklers {
     } yield { invocation =>
       invocation.data match
         case ItcMessage.Initialize(itcURI) =>
-          ItcClient.create[IO](itcURI, createClient) >>= (client => itcClient.complete(client).void)
+          for {
+            client <- ItcClient.create[IO](itcURI, createClient)
+            _      <- itcClient.complete(client).void
+            _      <- ITCVersionsRequests.queryItc[IO](cache, client).andWait(1.hour).foreverM.start
```

Comment:
So we could have up to an hour of using old itc version data, but the itc changes infrequently so it's not a big deal?

Reply:
Exactly this only hapeens when we change the itc data which doesn't change often. The server version changes on every lib update